src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationDashboard.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import styled from 'react-emotion';
+
+import AsyncView from 'app/views/asyncView';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import StackedBarChart from 'app/components/stackedBarChart';
+import {Panel, PanelBody, PanelHeader, PanelItem} from 'app/components/panels';
+import DateTime from 'app/components/dateTime';
+import LoadingError from 'app/components/loadingError';
+import EmptyMessage from 'app/views/settings/components/emptyMessage';
+
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {intcomma} from 'app/utils';
+import {t} from 'app/locale';
+import {SentryApp} from 'app/types';
+
+type Props = AsyncView['props'];
+
+type State = AsyncView['state'] & {
+  stats: any;
+  errors: any;
+  app: SentryApp | null;
+};
+
+export default class SentryApplicationDashboard extends AsyncView<Props, State> {
+  getDefaultState(): State {
+    return {
+      ...super.getDefaultState(),
+      stats: {},
+      errors: [],
+      app: null,
+    };
+  }
+
+  getEndpoints(): Array<[string, string, any] | [string, string]> {
+    const {appSlug} = this.props.params;
+    if (appSlug) {
+      // Default time range for now: 30 days ago to now
+      const now = Math.floor(new Date().getTime() / 1000);
+      return [
+        [
+          'stats',
+          `/sentry-apps/${appSlug}/stats/`,
+          {query: {since: now - 3600 * 24 * 30, until: now}},
```

Comment:
We should move the `3600 * 24 * 30` part into a variable too. Maybe named `one_month_ago` or something descriptive.